internal/rpcserver/interface.go

```diff
@@ -424,3 +424,43 @@ type Clock interface {
 	// Since returns the time elapsed since t.
 	Since(t time.Time) time.Duration
 }
+
+// FeeEstimator provides an interface that tracks historical data for published
+// and mined transactions in order to estimate fees to be used in new
+// transactions for confirmation within a target block window.
+//
+// The interface contract requires that all of these methods are safe for
+// concurrent access.
+type FeeEstimator interface {
+	// DumpBuckets returns the internal estimator state as a string.
+	DumpBuckets() string
+
+	// EstimateFee is the public version of estimateMedianFee. It calculates the
```

Comment:
I think `is the public version of estimateMedianFee` should be removed here, since the interface shouldn't need to necessarily change if the underlying implementation is updated to rely on a different internal function.